js/src/help-scout-beacon.js

```diff
@@ -78,10 +103,7 @@ function loadHelpScout( beaconId, sessionData = "" ) {
 
 	// eslint-disable-next-line new-cap
 	window.Beacon( "init", beaconId );
-	if ( sessionData !== "" ) {
-		// eslint-disable-next-line new-cap
-		window.Beacon( "session-data", JSON.parse( sessionData ) );
-	}
+	loadHelpScoutSessionData( sessionData );
```

Comment:
I fail to remember where this `sessionData` comes from. Is that a global that is set because of `window.Beacon()`?